src/doc/DocDrawingThread.js

```diff
@@ -78,8 +78,8 @@ class DocDrawingThread extends DrawingThread {
                 page: location.page,
                 dimensions: location.dimensions
             };
-            this.checkAndHandleScaleUpdate();
         }
+        this.checkAndHandleScaleUpdate();
```

Comment:
This should happen regardless of whether or not the thread page has been set already